src/views/partials/example/_macro.njk

```diff
@@ -1,7 +1,14 @@
 {% macro patternlibExample(params) %}
     {% set params = helpers.generateExampleParams(params, helpers.addDependency) %}
     <div class="patternlib-example js-example">
-        <a href="{{ params.link }}" target="_blank" class="patternlib-example__link u-fs-s">Open this example in a new window</a>
+        <div class="patternlib-example__header">
+            <a href="{{ params.link }}" target="_blank" class="patternlib-example__header-link external-link u-fs-s">Open this example in a new tab
+            <svg id="external-link" class="svg-icon" viewBox="0 0 12 12" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Should us the external link macro here instead but will also then need to change that macro to allow classes to be set